common/lib/xmodule/xmodule/video_module/video_xfields.py

```diff
@@ -173,3 +173,12 @@ class VideoFields(object):
         scope=Scope.settings,
         default="",
     )
+    date_last_view_bumper = DateTime(
+        display_name=_("Date of the last view of the bumper"),
+        scope=Scope.preferences,
+    )
+    do_not_show_again_bumper = Boolean(
```

Comment:
Fixed: «do_not_show_again_bumper to bumper_do_not_show_again"

--
Sincerely,  
Alexander Kryklia
cell: +38 096 722 55 46
skype: auraz_
Отправлено при помощи Sparrow (http://www.sparrowmailapp.com/?sig)

Sent with Sparrow (http://www.sparrowmailapp.com/?sig)  

On Thursday, May 21, 2015 at 14:43, Usman Khalid wrote:

> In common/lib/xmodule/xmodule/video_module/video_xfields.py (https://github.com/edx/edx-platform/pull/7825#discussion_r30794019):
> > @@ -173,3 +173,12 @@ class VideoFields(object): > scope=Scope.settings, > default="", > ) > + date_last_view_bumper = DateTime( > + display_name=_("Date of the last view of the bumper"), > + scope=Scope.preferences, > + ) > + do_not_show_again_bumper = Boolean(  
> And since we can never really update field names, can we at the minimum change do_not_show_again_bumper to bumper_do_not_show_again? This would kind of namespace the two fields with bumper_.
>  
> —
> Reply to this email directly or view it on GitHub (https://github.com/edx/edx-platform/pull/7825/files#r30794019).
>  
>  
>  
>  
>